src/Dropdown.php

```diff
@@ -2462,7 +2465,10 @@ public static function showGlobalSwitch($ID, $attrs = [])
     public static function import($itemtype, $input)
     {
 
-        if (!($item = getItemForItemtype($itemtype))) {
+        if (
+            !($item = getItemForItemtype($itemtype))
+            || !method_exists($item, 'import')
```

Comment:
Correct me if I'm wrong, but we must have a an instance of `Dropdown` or `CommonDropdown`; and methods `import` and `importExternal` does exists.
Checking `$item` has the expected type is probably a better solution.